src/index.js

```diff
@@ -109,31 +109,47 @@ function drawGame(gameState, myID) {
   let diceCountDiv = doc.createElement('div');
   diceCountDiv.className = 'diceCount';
   gameDiv.appendChild(diceCountDiv);
-  diceCountDiv.textContent = "There are  " + gameState.diceTotal + " dice in the game";  
+  diceCountDiv.textContent = "There are  " + gameState.diceTotal + " dice in the game"; 
   for (let i = 0; i < userCount; i++) {
     let roll = gameState.users[i].dice;
     let name = gameState.users[i].name;
-    let player = doc.createElement('div');
-    if (gameState.users[i].dice.length == 0) {
-      player.className = 'explayer';
-      player.textContent = name + " is out of the game";  
-    } else {
-      player.className = 'player';
-      player.textContent = name + ":  ";
-      let imageRoll = getImages(roll);
-      for (let i = 0; i < imageRoll.length; i++) {
+    let users = gameState.users;
+    let player = doc.createElement('div');   
+    if (gameState.users[i].dice.length > 0) { 
+      if (i == 1) { 
+        player.className = 'it';
```

Comment:
All the common stuff here can be executed before the conditional, then the conditional can just determine which of the different lines should be executed.